packages/hmr/reactivity/functional.py

```
@@ -0,0 +1,54 @@
+from collections.abc import Callable
+from functools import wraps
+from typing import Protocol, overload
+
+from .helpers import Memoized, MemoizedMethod, MemoizedProperty
+from .primitives import Batch, Effect, Signal
+
+
+class Getter[T](Protocol):
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (reactivity.functional, line 9)'

(E0001)